vendor/assets/javascripts/behaviors/sortable.js

```diff
@@ -3,6 +3,7 @@ Cmsify.Sortable = function(el, options) {
   this.options = options;
   var sortableElement = sortable(this.$el, this.options);
   sortableElement[0].addEventListener('sortupdate', function(e) {
+    $('input[type=submit]').prop('disabled', false);
```

Comment:
Yeah, I couldn't figure out a way to do it, I tried `'mouseup'`, `'mousedown'` and `'click'` all on `$(window)` but none of them would register until the second change had been made.